extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -18,15 +18,16 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 	window.googletag = window.googletag || {};
 	window.googletag.cmd = window.googletag.cmd || [];
 
+	function isSlotRegistered(slot, id) {
+		return registeredCallbacks[id] && slot && slot === googleSlots.getSlot(id);
```

Comment:
@fraszczakszymon please, check it

it's safer to check `slot`, `undefined === undefined`